src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -271,6 +277,13 @@ public void DisplayHelp()
                 _window.Write(HelpIndent);
                 _window.WriteLine(line);
             }
+
+            _window.WriteLine(InteractiveWindowResources.ScriptDirectives);
+            foreach (var line in s_scriptDirectives)
+            {
+                _window.Write(HelpIndent);
+                _window.WriteLine(line);
+            }
```

Comment:
Will we print this for python as well? We only want to print these for C# and VB interactive window as it will not make any sense for Python or anyone else using the same Interactive window.